nav2_bringup/launch/bringup_launch.py

```diff
@@ -27,7 +27,7 @@
 from launch.launch_description_sources import PythonLaunchDescriptionSource
 from launch.substitutions import LaunchConfiguration, PythonExpression
 from launch_ros.actions import Node
-from launch_ros.actions import PushROSNamespace
+# from launch_ros.actions import PushROSNamespace
```

Comment:
I think the name just changed from `PushRosNamspace` to `PushROSNamespace`.  I think can use the former